pandas/tests/groupby/aggregate/test_aggregate.py

```diff
@@ -313,3 +313,14 @@ def test_order_aggregate_multiple_funcs():
     expected = pd.Index(['sum', 'max', 'mean', 'ohlc', 'min'])
 
     tm.assert_index_equal(result, expected)
+
+
+def test_uint64_type_handling():
```

Comment:
can you add a battery of tests in pandas/tests/dtypes/test_common.py I don't think we have this tested at all.

Reply:
I did add the parameterization as asked below. I wasn't sure if you meant I should add new tests in addition to `test_uint64_type_handling`